cmd/atlascmd/schema.go

```diff
@@ -334,3 +339,15 @@ func fmtFile(task fmttask) (bool, error) {
 	}
 	return false, nil
 }
+
```

Comment:
Why are you not using https://pkg.go.dev/github.com/spf13/pflag?utm_source=godoc#StringToStringVarP?

Reply:
nice